### ISSUE TITLE
ibconfig: 1.7.2 -> 1.7.3 (#127633) causes sslh to be unable to parse configuration files

### ISSUE THREAD

**Issue by jcumming** [NixOS/nixpkgs#129450]
**Describe the bug**

As seen [upstream](https://github.com/yrutschle/sslh/issues/292) libconfig-1.7.3 removed some functionality which causes sslh to fail to parse config files with the rather cryptic error message:

```
At least one target protocol must be specified.
```

**To Reproduce**
1. install sslh from master
2. run it with the included `basic.cfg`

**Expected behavior**
Should function as it did with `libconfig-1.7.2`

**Notify maintainers**
@koral, @fpletz 
This is fixed [upstream](https://github.com/yrutschle/sslh/commit/24e7f46a431b59043bf534eb7cdab76c4ffd595a) but has not been released. The fix does not apply (easily) to the latest released version. 

Maintainer information:
```yaml
# a list of nixpkgs attributes affected by the problem
attribute:
# a list of nixos modules affected by the problem
module:
```

**Comment by JohnAZoidberg** [comment]
Might be fixed by #138614 according to the release notes: https://github.com/yrutschle/sslh/blob/3fb1201b3fbd7a3f8ba743a002b6d29c424abc0f/ChangeLog#L28

**Comment by JohnAZoidberg** [comment]
Works on my system with the latest unstable that includes the PR I mentioned.
Please reopen if it's not the case for you.
